packages/toolkit/src/query/core/buildMiddleware/polling.ts

```diff
@@ -102,10 +106,7 @@ export const build: SubMiddlewareBuilder = ({
       const currentPoll = currentPolls[queryCacheKey]
```

Comment:
This line can move under the next block now.

Reply:
👍